client/src/call-customer/components/clock.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Component names should be Uppercase.
This should be applied also for get-tciket components (missed it)